simulator/simulator_app/src/main.cpp

```diff
@@ -7,7 +7,7 @@ int main(int /*argc*/, const char* /*argv*/[])
 {
     using namespace std::chrono_literals;
 
-    const StringPtr configPath = "/tmp/opendaq-config.json";
+    const StringPtr configPath = "/home/opendaq/opendaq/opendaq-config.json";
```

Comment:
What if you want to build the simulator on Windows? There is no `/home/` dir.

Reply:
Is it included in any Windows build / configuration? I thought that this is only used by the CI as a separate simulator build step.